### ISSUE TITLE
Warning 'Each child in an array or iterator should have a unique "key" prop'

### ISSUE THREAD

**Issue by igisav** [CassetteRocks/react-infinite-scroller#133]
Using infinite scroller throw error(warning) in console with 'Each child in an array or iterator should have a unique "key" prop'.

If I open simple example from `docs/src/index.js` from this project it's works fine. But when I use the same example in my project, it's throws this error, but only with version 1.1.3.
I'm using [Create React App](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md) and [material ui](https://github.com/mui-org/material-ui)

Something wrong between v1.1.2 and v1.1.3. I assume,it's commit 0db28b6 .

**Comment by danbovey** [comment]
Yep, I probably should have bumped it up to `2.0.0` as it was a slightly breaking change for people who use custom loaders.

The [release notes for 1.1.3](https://github.com/CassetteRocks/react-infinite-scroller/releases/tag/1.1.3) link to this comment:

https://github.com/CassetteRocks/react-infinite-scroller/pull/127#issuecomment-357910337

**Comment by cutie6** [comment]
> Yep, I probably should have bumped it up to `2.0.0` as it was a slightly breaking change for people who use custom loaders.
> 
> The [release notes for 1.1.3](https://github.com/CassetteRocks/react-infinite-scroller/releases/tag/1.1.3) link to this comment:
> 
> [#127 (comment)](https://github.com/danbovey/react-infinite-scroller/pull/127#issuecomment-357910337)
> 
> TL;DR: Add `key={0}` to your loader component.

takes me some time to find this answer to that werid warning.........
